lib/Echidna/Onchain.hs

```diff
@@ -76,8 +77,8 @@ data FetchedContractData = FetchedContractData
   }
   deriving (Generic, ToJSON, FromJSON, Show)
 
-instance ToJSONKey W256 where
-  toJSONKey = toJSONKeyText (Text.pack . show)
+--instance ToJSONKey W256 where
+--  toJSONKey = toJSONKeyText (Text.pack . show)
```

Comment:
Please drop this if it is no longer needed

Reply:
Actually, hevm used some instance (and therefore, I had to comment it). I'm not sure if this affects our code or not 🤔